cmd/todo/todo.go

```diff
@@ -0,0 +1,45 @@
+package main
+
+import (
+	"log"
+	"net/http"
+	"os"
+
+	"github.com/joho/godotenv"
+	"github.com/yandex-go-master/go_final_project/internal/database"
+	"github.com/yandex-go-master/go_final_project/internal/handlers"
+	_ "modernc.org/sqlite"
+)
+
+func main() {
+	log.Println("INFO: starting up...")
+
+	err := godotenv.Load()
+	if err != nil {
+		log.Fatal("ERR: cannot load .env file")
```

Comment:
Теряешь ошибку, можешь потом много времени потратить на поиск проблем
```suggestion
		log.Fatal("ERR: cannot load .env file: %v", err)
```